source/classes/Archive.js

```diff
@@ -66,6 +66,17 @@
 	};
 
 	/**
+	 * Whether or not this archive has a group with the given title.
+	 * @param {String} The group's title
+	 * @returns {true|false}
+	 */
+	Archive.prototype.hasGroup = function(groupTitle) {
```

Comment:
Perhaps using the name `containsGroupWithTitle` would be better suited. My reasoning:
 * containsGroup (vs hasGroup) I feel better reflects the recursiveness of the method (hasGroup sounds shallow)
 * I'd like to use `containsGroup` for searching with an actual group

What do you think?